### PR TITLE
manpages rewritten/converted in ReStructuredText

### PR DESCRIPTION
**Pull request by speters**
Basically the same as https://github.com/openv/vcontrold/pull/10

This branch features
 * rewritten or newly written manual pages in ReStructuredText format
 * conversion of vcontrold manpage to ReStructuredText
 * conversion to "standard unix" troff man format using [rst2man](http://docutils.sourceforge.net/)

There are some minor hiccups, as e.g.
 * a table in vclient.1 rendered with an empty line between 1st and 2nd row
 * option lists required some weird escaping of - to \\- - I could not spot exactly why

### CODE REVIEW

**Comment by l3u** [comment]
Nice work :-) I only did a little bit of formatting (as I'm the formatting guy here ;-)

I also changed the variable names. I think it's always meaningful to choose expressive ones (just a suggestion! If you don't like it, revert it!). If you want, you can read through https://community.kde.org/Policies/Kdelibs_Coding_Style for the style we use e. g. at KPhotoAlbum and I personally also use for other projects. Most of it is really meaningful and handy.

File: doc/man/CMakeLists.txt [hunk 1]
```diff
@@ -1,29 +1,29 @@
-# generate classic *roff manual pages from markdown
-# this requires md2man (https://github.com/sunaku/md2man)
+# Generate classic *roff manual pages from ReStructuredText.
+# This requires Python's Docutils (http://docutils.sourceforge.net/)
 
 cmake_minimum_required(VERSION 3.2.0)
```
Comment (l3u) [inline doc/man/CMakeLists.txt]: Do we have to re-declare this here? It's already in the main `CMakeLists.txt`?

File: CMakeLists.txt [hunk 2]
```diff
@@ -58,9 +58,9 @@ add_dependencies(vclient UpdateVersion)
 add_executable(vsim ${vsim_SRCS})
 add_dependencies(vsim UpdateVersion)
 
+add_subdirectory(${CMAKE_CURRENT_SOURCE_DIR}/doc/man)
```
Comment (l3u) [inline CMakeLists.txt]: Sure you want to add this as a subdirectory and not directly here? I would add it to the main `CMakeLists.txt` as it's not too much code. Would be still be overseeable imo.

Comment (speters) [reply CMakeLists.txt]: IMO this modularity comes at no cost in this case and makes changes to the docs clearly separateable from the source codes.

**Comment by speters** [comment]
Thx for all those fixes and helpful comments!

File: CMakeLists.txt (hunk 2 above)
Comment (l3u) [reply CMakeLists.txt]: Let every man seek heaven in his own fashion ;-)

**Comment by speters** [comment]
Merged into master.

**Pull request closed without merging**